ios/BluedotPointSDK.m

```diff
@@ -19,9 +19,13 @@ - (instancetype)init {
     self = [super init];
     if (self) {
         
+        // Deprecated
         BDLocationManager.instance.sessionDelegate = self;
```

Comment:
@toroduque you can remove sessionDelegate entirely

Reply:
Removed!